app/models/framework/management_charge_calculator/column_based.rb

```diff
@@ -12,6 +12,15 @@ def calculate_for(entry)
         column_value_for_entry = entry.data.dig(column).to_s.downcase
         percentage = value_to_percentage[column_value_for_entry]
 
+        if percentage.nil?
+          Rollbar.error(
+            "Got value '#{column_value_for_entry}' for '#{column}' on #{entry.framework.short_name}"\
```

Comment:
Observation: this will not tell us the case (casing?) of the string as it was put in, since we are not preserving the value of the input before downcasing it.